NuCivic/dkan
# Added validation to field data dictionary with pod strict enable

Issue: https://jira.govdelivery.com/browse/CIVIC-5525

## Description

We need to add validation to field data dictionary when pod strict mode is enable.

## How to reproduce

1.  Enable pod strict mode.
2. Try to create a dataset with simple string on data dictionary.
3. You don't have any.

## QA Steps

- [ ] Enable pod strict mode.
- [ ] Try to create a dataset with field data dictionary value (test or another simple text). You should see a error mesage.
- [ ] Try to create a dataset with field data dictionary value (Empty, url, redacted) and you shouldn't see any error



## Reminders
